app/brave_main_delegate.cc

```diff
@@ -71,7 +72,14 @@ bool BraveMainDelegate::ShouldEnableProfilerRecording() {
   return false;
 }
 
+void BraveMainDelegate::DisableAPIs() {
```

Comment:
we should be calling these in `ContentRendererClient::
    SetRuntimeFeaturesDefaultsBeforeBlinkInitialization()` in `BraveContentRendererClient`